### PR TITLE
fix(passport): Passport template - tweaks and fixes

### PR DESCRIPTION
**Pull request by albinagu**
## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/passport/src/forms/Done.ts [hunk 1]
```diff
@@ -15,7 +15,11 @@ export const Done: Form = buildForm({
     buildMultiField({
       id: 'done',
       title: m.applicationComplete,
-      description: m.applicationCompleteDescription,
+      description: (application: Application) =>
+        `Umsókn þín um vegabréf fyrir **${
+          (application.answers.childsPersonalInfo as ChildsPersonalInfo)
+            ?.name ?? (application.answers.personalInfo as any)?.name
+        }** hefur verið móttekin.`,
```
Comment (darrikonn) [inline libs/application/templates/passport/src/forms/Done.ts:22]: Shouldn't this support translations as well?

File: libs/application/templates/passport/src/forms/ParentB.ts [hunk 2]
```diff
@@ -32,18 +30,15 @@ export const ParentB: Form = buildForm({
...
-            ' ' +
-            format(new Date(application.created), 'dd.MMMM, yyyy', {
-              locale: localeIS,
-            }) +
-            '. ' +
-            m.parentBIntroPart2.defaultMessage,
+            `**${
+              (application.answers.childsPersonalInfo as ChildsPersonalInfo)
+                .guardian1.name
+            }** um vegabréf fyrir **${
```
Comment (darrikonn) [inline libs/application/templates/passport/src/forms/ParentB.ts:38]: ditto

File: libs/application/templates/passport/src/forms/WaitingForParentBConfirmation.ts [hunk 3]
```diff
@@ -14,8 +15,22 @@ export const WaitingForParentBConfirmation: Form = buildForm({
...
       title: m.waitingForConfirmationTitle,
-      description: m.waitingForConfirmationDescription,
-      children: [buildDividerField({ title: ' ' })],
+      description: (application: Application) =>
+        `Umsókn þín um vegabréf fyrir **${
+          (application.answers.childsPersonalInfo as ChildsPersonalInfo)?.name
+        }** hefur verið send til **${
+          (application.answers.childsPersonalInfo as ChildsPersonalInfo)
+            ?.guardian2.name
+        }** til samþykktar.`,
```
Comment (darrikonn) [inline libs/application/templates/passport/src/forms/WaitingForParentBConfirmation.ts:24]: ditto

File: libs/application/templates/passport/src/lib/messages.ts [hunk 4]
```diff
@@ -317,25 +317,26 @@ export const m = defineMessages({
   /* Waiting For Confirmation Section */
   waitingForConfirmationTitle: {
     id: 'pa.application:waitingForConfirmation.title',
-    defaultMessage: 'Title',
+    defaultMessage: 'Umsókn staðfest',
```
Comment (darrikonn) [inline libs/application/templates/passport/src/lib/messages.ts:320]: ditto

File: libs/application/templates/passport/src/lib/messages.ts [hunk 5]
```diff
@@ -317,25 +317,26 @@ export const m = defineMessages({
...
     description: 'Some description',
   },
   waitingForConfirmationDescription: {
-    id: 'pa.application:waitingForConfirmation.description',
-    defaultMessage: 'Description',
+    id: 'pa.application:waitingForConfirmation.description#markdown',
+    defaultMessage:
+      'Umsókn þín um vegabréf fyrir **{childsName}** hefur verið send til **{guardian2Name}** til samþykktar.',
```
Comment (darrikonn) [inline libs/application/templates/passport/src/lib/messages.ts:326]: ditto on the changes in this file (below as well)

**Review by alexdiljar — APPROVED** [review]

**Review by darrikonn — APPROVED** [review]

**Pull request merged**